packages/CodingStandard/src/Fixer/LineLength/BreakArrayListFixer.php

```diff
@@ -86,12 +60,9 @@ public function fix(SplFileInfo $file, Tokens $tokens): void
                 continue;
             }
 
-            $arrayWrapper = $this->arrayWrapperFactory->createFromTokensArrayStartPosition($tokens, $position);
-            if ($arrayWrapper->isAssociativeArray()) {
-                continue;
-            }
+            [$blockStart, $blockEnd] = $this->blockStartAndEndFinder->findInTokensByBlockStart($tokens, $position);
```

Comment:
In quite a few places I see a function returning a pair like this and then those values are pretty much always both passed to another method. It might be worth adding a value object holding these two values. What do you think?

Reply:
Very good idea! Thanks